buildSrc/src/main/groovy/org/apache/geode/gradle/plugins/DependencyConstraints.groovy

```diff
@@ -83,76 +83,76 @@ class DependencyConstraints implements Plugin<Project> {
     project.dependencies {
       constraints {
         // informal, inter-group dependencySet
-        api('com.sun.xml.bind:jaxb-impl:2.3.1')
+        api('com.sun.xml.bind:jaxb-impl:2.3.2')
         api('javax.xml.bind:jaxb-api:2.3.1')
 
         api(group: 'antlr', name: 'antlr', version: get('antlr.version'))
         api(group: 'cglib', name: 'cglib', version: get('cglib.version'))
-        api(group: 'com.carrotsearch.randomizedtesting', name: 'randomizedtesting-runner', version: '2.5.0')
+        api(group: 'com.carrotsearch.randomizedtesting', name: 'randomizedtesting-runner', version: '2.7.3')
         api(group: 'com.fasterxml.jackson.module', name: 'jackson-module-scala_2.10', version: '2.10.0')
         api(group: 'com.github.davidmoten', name: 'geo', version: '0.7.1')
         api(group: 'com.github.stefanbirkner', name: 'system-rules', version: '1.19.0')
         api(group: 'com.github.stephenc.findbugs', name: 'findbugs-annotations', version: '1.3.9-1')
-        api(group: 'com.google.guava', name: 'guava', version: '27.0-jre')
+        api(group: 'com.google.guava', name: 'guava', version: '28.1-jre')
         api(group: 'com.google.protobuf', name: 'protobuf-gradle-plugin', version: get('protobuf-gradle-plugin.version'))
         api(group: 'com.google.protobuf', name: 'protobuf-java', version: get('protobuf-java.version'))
         api(group: 'com.healthmarketscience.rmiio', name: 'rmiio', version: '2.1.2')
-        api(group: 'com.mockrunner', name: 'mockrunner-servlet', version: '1.1.2')
+        api(group: 'com.mockrunner', name: 'mockrunner-servlet', version: '2.0.1')
         api(group: 'com.sun.activation', name: 'javax.activation', version: '1.2.0')
-        api(group: 'com.sun.istack', name: 'istack-commons-runtime', version: '2.2')
-        api(group: 'com.tngtech.archunit', name:'archunit-junit4', version: '0.10.2')
-        api(group: 'com.zaxxer', name: 'HikariCP', version: '3.2.0')
-        api(group: 'commons-beanutils', name: 'commons-beanutils', version: '1.9.3')
+        api(group: 'com.sun.istack', name: 'istack-commons-runtime', version: '3.0.9')
+        api(group: 'com.tngtech.archunit', name:'archunit-junit4', version: '0.11.0')
```

Comment:
We're upgrading major versions of dependencies here, in the context of a minor of Geode. If we emit object from the packages as part of our API, then we might be breaking downstream clients.

Reply:
I think these should be safe to upgrade.